storage/src/tests/storageapi/mbusprot/mbusprot.4.2.serialization.V_4_2_STABLE

```diff
@@ -1,66 +0,0 @@
-
```

Comment:
@vekterli Was removal of this file intentional?

Reply:
It's intentional, although it could've/should've been its own micro-PR; I just did it here since I stumbled across it as part of adding 2-phase RemoveLocation to the storage API protocol. It's an ancient leftover file from the days where we used mbus for the StorageAPI and had a daisy-chain of protocol versions supported. The "4.2" version part indicates it was practically museum-ready anyway 🦖